messages/en.json

```diff
@@ -3,13 +3,18 @@
     "home": "Home"
   },
   "customer_case": {
-    "customer": "Customer",
+    "customer": "The customer",
     "project": "Project",
     "duration": "Duration",
     "industry": "Industry",
     "delivery": "Delivery",
-    "consultants": "Consultants",
-    "in_project": "In project"
+    "varianter": "Varianter",
```

Comment:
Should it be "Variants" instead of "Varianter" (even though we don't have a good translation for this yet) 😅